### DISCUSSION TITLE
Two roasters simultaneously?

### DISCUSSION THREAD

**Discussion by russ999** [artisan-roaster-scope/artisan#707] [Q&A] [answered]
How can artisan be used to roast on two roasters simultaneously?  Right now I roast very happily on one modified/home built air roaster with 4 k-type probes, Center 304 usb thermometer, manual heat/motor controls, and artisan 2.4.2 plotting temperatures on a MacBook Air with a UART bridge.  I hope to add a second air roaster (I want more coffee!) using two probes for each roaster. But I can’t seem to get artisan to open two windows or run two instances. I even installed an older version of artisan and I can run both versions simultaneously but can’t get them to both read the Usb port at the same time. I have a second usb thermometer (Center 302, two probes) but can’t seem to get a second port or bridge running.  Do I need two laptops?

**Comment by MAKOMO** [comment, marked as answer]
You need a second laptop

**Comment by danneblo** [comment]
Hi! Is this still the case or is it possible to connect two roasters to one computer?

**Reply by MAKOMO** [reply]
As said, you need a second laptop. This will never change.
